### PR TITLE
typography: Monthly good

### PR DESCRIPTION
**Pull request by ryan-ludwig**
Covers the monthly good flow

https://test.kiva.org/monthlygood/
https://test.kiva.org/monthlygood/setup?amount=25&category=default
https://test.kiva.org/monthlygood/thanks

https://www.kiva.org/monthlygood/
https://www.kiva.org/monthlygood/setup?amount=25&category=default
https://www.kiva.org/monthlygood/thanks

May want to pull down this branch and do testing on your VM to go through the whole flow.

### CODE REVIEW

File: src/pages/MonthlyGood/HowItWorks.vue [hunk 1]
```diff
@@ -1,58 +1,78 @@
...
-			<h1 class="column small-12 tw-text-center impact-text">
-				How it works
-			</h1>
-
-			<div class="small-12 large-4 column step-wrapper tw-text-center">
-				<icon-auto-deposit-alternate class="mg-image" />
-				<h4>Contribute automatically</h4>
-				<div class="description-text">
-					<p>Sign up once by choosing a monthly amount that works for you. Cancel anytime.</p>
+	<kv-theme-provider :theme="darkGreenTheme">
```
Comment (mcstover) [inline src/pages/MonthlyGood/HowItWorks.vue:2]: Nice use of the theme!

**Review by mcstover — APPROVED** [review]

File: src/components/MonthlyGood/AlreadySubscribedNotice.vue [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 	<div v-else>
-		<h2 class="tw-text-center">
+		<h2 class="tw-text-center tw-mb-2">
 			You're signed up for Monthly Good, so we cannot process a one-time contribution on your account.
 		</h2>
-
-		<p class="tw-text-center">
+		<p class="tw-text-center tw-mb-2">
 			<!-- eslint-disable-next-line max-len -->
 			Instead, you can change your <a href="/settings/subscriptions">subscription settings</a> to direct your Monthly Good contribution to COVID-19 support loans?
```
Comment (emuvente) [inline src/components/MonthlyGood/AlreadySubscribedNotice.vue:19]: Should this also be a router-link?

**Review by emuvente — APPROVED** [review]
Awesome!

**Review by eddieferrer — APPROVED** [review]

**Comment by eddieferrer** [comment]
This looks awesome, the pages look so much better

**Pull request merged**